rj_gameplay/stp/skill/cost_behavior.py

```diff
@@ -35,8 +37,10 @@ def update(self) -> py_trees.common.Status:
         Creates new move action to get robot to move there
         Always returns running
         """
-        move = Move.Move(np.array([0,1+self.robot.id]))
-        move.tick(self.robot)
+        # self.move = move.Move(target_point = np.array([0,1+self.robot.id]))
+        # self.move.tick(self.robot)
+        self.move = move.Move(self.robot.id, target_point = np.array([0.0,1.0+self.robot.id]))
```

Comment:
This should only set the target point, not create a new action.

Reply:
CostBehavior is kinda broken right now, so I didn't end up using this code. I'll revert it back to what it was originally.